src/wap_client.c

```diff
@@ -192,6 +192,16 @@ prepare_save_command (client_t *self)
 }
 
 
+//  ---------------------------------------------------------------------------
+//  prepare_start_command
+//
+
+static void
+prepare_start_command (client_t *self)
+{
+    wap_proto_set_start_height (self->message, self->args->start_height);
+}
+
```

Comment:
This is hand written. If's how the client prepares the START command: takes
the corresponding method arguments and stores them in the message.

On Sun, Jan 4, 2015 at 4:27 PM, oranjuice <notifications@github.com> wrote:

> In src/wap_client.c
> <https://github.com/oranjuice/monero-0mq/pull/4#discussion-diff-22439998>:
>
> > @@ -192,6 +192,16 @@ prepare_save_command (client_t *self)
> >  }
> >
> >
> > +//  ---------------------------------------------------------------------------
> > +//  prepare_start_command
> > +//
> > +
> > +static void
> > +prepare_start_command (client_t *self)
> > +{
> > +    wap_proto_set_start_height (self->message, self->args->start_height);
> > +}
> > +
>
> was this generated code? This function doesn't seem to appear for me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oranjuice/monero-0mq/pull/4/files#r22439998>.
>